PRINCIPLES.md

```diff
@@ -10,22 +10,22 @@ These principles were derived from modern software operations but are rooted in
 
 ## Principles
 
-1. **The principle of declarative desired state**
+1. **Desired System State Is Declarative**
 
     A system managed by GitOps must have its _Desired State_ expressed declaratively as data in a format writable and readable by both humans and machines.
 
-2. **The principle of immutable desired state versions**
+2. **Declarations Are Versioned and Immutable**
```

Comment:
```suggestion
2. **Declarations are Versioned and Immutable**
```
Typographic Suggestion:
`Are` -> `are`
to align with
https://user-images.githubusercontent.com/407675/126000898-eb81abd1-a947-40e4-b313-107fd45b7ad7.png
Thanks @scottrigby

Reply:
@lloydchang Yeah I wasn't sure about which title case style we want to adopt. I would recommend not getting too caught up in this question, and that we follow some existing style guide. For now I just chose AP style [AP style Title Case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case) so I didn't have to think about it.

If interested though, this is a fun tool to see how the different standards look https://titlecaseconverter.com/